fs/operations/rc.go

```diff
@@ -224,9 +224,10 @@ func rcSingleCommand(ctx context.Context, in rc.Params, name string, noRemote bo
 			return nil, err
 		}
 		autoFilename, _ := in.GetBool("autoFilename")
+		autoFilenameHeader, _ := in.GetBool("autoFilenameHeader")
```

Comment:
If we decide to keep this flag, then it needs documenting above in the `copyurl` docs.